packages/headless/src/features/commerce/product-templates/product-templates-manager.ts

```diff
@@ -1,31 +1,26 @@
-import {ProductRecommendation} from '../../../api/search/search/product-recommendation';
+import {Product} from '../../../api/commerce/common/product';
```

Comment:
@fbeaudoincoveo @olamothe I'm not too use what this is used for but I had to change the type from `ProductRecommendation` to the new `Product`